text/0265-event-vision.md

```diff
@@ -0,0 +1,72 @@
+# Event basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to write down and agree on a few basics.
+
+### What are events?
+
+Events are a type of log which has a required event name and a specific structure which is implied by that event name.
+
+The event name and its implied structure make events much more suitable for observability compared to traditional logs.
+
+Because of this, the OpenTelemetry project wants to encourage the transition from traditional logs to events.
+
+### OTLP
+
+Since events are a type of log, they share the same OTLP data structure and OTLP pipeline.
+
+### API
+
+OpenTelemetry should have an Event API. This will help to promote the distinction between events and traditional logs,
+and encourage the use of events over traditional logs.
+
+### Interoperability with traditional logs
+
+It should be possible to send events from the Event API to a traditional logging library (e.g. Log4j).
+This allows users to integrate events from the Event API into an existing (non-OpenTelemetry) log stream.
+
+Note: If a user chooses to send events from the Event API to a traditional logging library, and they have
```

Comment:
Yeah, I'd be happy making this stronger or even "ban" (or more strongly word) this scenario...

While I can see (if I squint a LOT) why someone would want to do this, but adding wording to say and any Event SDK (or processor) that does this MUST NOT populate the `event.name` attribute "to avoid" the duplication as called out here. Which yes would lead to `events` JUST becoming (and treated as) standard unstructured log records -- which may be fine for some backends / use cases.

Reply:
I don't think we want to drop the `event.name` since it's such a critical part of event definition